cstore_fdw.c

```diff
@@ -1506,29 +1336,18 @@ ValidateForeignTableOptions(char *filename, char *compressionTypeString,
 									BLOCK_ROW_COUNT_MAXIMUM)));
 		}
 	}
-}
-
-
-/*
- * CStoreDefaultFilePath constructs the default file path to use for a cstore_fdw
- * table. The path is of the form $PGDATA/cstore_fdw/{databaseOid}/{relfilenode}.
- */
-static char *
-CStoreDefaultFilePath(Oid foreignTableId)
-{
-	Relation relation = relation_open(foreignTableId, AccessShareLock);
-	RelFileNode relationFileNode = relation->rd_node; 
 
-	Oid databaseOid = relationFileNode.dbNode;
-	Oid relationFileOid = relationFileNode.relNode;
-
-	StringInfo cstoreFilePath = makeStringInfo();
-	appendStringInfo(cstoreFilePath, "%s/%s/%u/%u", DataDir, CSTORE_FDW_NAME,
-					 databaseOid, relationFileOid);
-
-	relation_close(relation, AccessShareLock);
-
-	return cstoreFilePath->data;
+	/* check if the provided logging flag is valid */
+	if (loggingString != NULL)
+	{
+		if (strncmp(LOGGING_STRING_TRUE, loggingString, NAMEDATALEN) != 0 &&
+			strncmp(LOGGING_STRING_FALSE, loggingString, NAMEDATALEN) != 0)
```

Comment:
parse_pool accepts variety of input values. I am trying to restrict this option specifically to "true" or "false"